hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/testing.md

```diff
@@ -1274,6 +1282,21 @@ bin/hadoop s3guard markers -clean -verbose $BUCKET
 # expect success and exit code of 0
 bin/hadoop s3guard markers -audit -verbose $BUCKET
 
+# ---------------------------------------------------
+# Copy to from local
+# ---------------------------------------------------
+
+time bin/hadoop fs -copyFromLocal -t 10  share/hadoop/tools/lib/*aws*jar $BUCKET/
+
+# expect the iostatistics object_list_request value to be O(directories)
+bin/hadoop fs -ls -R $BUCKET/
+
+# expect the iostatistics object_list_request and op_get_content_summary values to be 1 
```

Comment:
Yes there is a extra space here at the end as pointed by Yetus,

Reply:
fixed that last night; will push and then merge